sources/api/netdog/src/main.rs

```diff
@@ -371,6 +382,50 @@ fn set_hostname(args: SetHostnameArgs) -> Result<()> {
     Ok(())
 }
 
+/// Set and apply default sysctls for the primary network interface
+fn prepare_primary_interface() -> Result<()> {
+    let primary_interface =
+        fs::read_to_string(PRIMARY_INTERFACE).context(error::PrimaryInterfaceReadSnafu {
+            path: PRIMARY_INTERFACE,
+        })?;
+    write_interface_sysctl(primary_interface, PRIMARY_SYSCTL_CONF)?;
+
+    // Execute `systemd-sysctl` with our configuration file to set the sysctls
+    let systemd_sysctl_result = Command::new(SYSTEMD_SYSCTL)
+        .arg(PRIMARY_SYSCTL_CONF)
+        .output()
+        .context(error::SystemdSysctlExecutionSnafu)?;
+    ensure!(
+        systemd_sysctl_result.status.success(),
+        error::FailedSystemdSysctlSnafu {
+            stderr: String::from_utf8_lossy(&systemd_sysctl_result.stderr)
+        }
+    );
+    Ok(())
+}
+
+/// Write the default sysctls for a given interface to a given path
+fn write_interface_sysctl<S, P>(interface: S, path: P) -> Result<()>
+where
+    S: AsRef<str>,
+    P: AsRef<Path>,
+{
+    let interface = interface.as_ref();
+    let path = path.as_ref();
+    // TODO if we accumulate more of these we should have a better way to create than format!()
+    // Accept router advertisement (RA) packets even if IPv6 forwarding is enabled on interface
+    let ipv6_accept_ra = format!("net.ipv6.conf.{}.accept_ra = 2", interface);
+    // Enable loose mode for reverse path filter
+    let ipv4_rp_filter = format!("net.ipv4.conf.{}.rp_filter = 2", interface);
```

Comment:
Given the potentially dire consequences of `systemd-sysctl` failing to return successfully, we might want to allow failure here:
```suggestion
    // Accept router advertisement (RA) packets even if IPv6 forwarding is enabled on interface
    let ipv6_accept_ra = format!("-net.ipv6.conf.{}.accept_ra = 2", interface);
    // Enable loose mode for reverse path filter
    let ipv4_rp_filter = format!("-net.ipv4.conf.{}.rp_filter = 2", interface);
```